aeneas/src/x86/X86CodeGen.v3

```diff
@@ -379,8 +453,10 @@ class X86CodeGen extends MachCodeGen {
 		var ptr = i.inputs[0], ssaval = i.input1();
 		if (SsaConst.?(ptr.dest)) {
 			// store to absolute address
+			var t = i.op.typeArgs[1];
 			var addr = Addr.!(val(ptr.dest));
-			gen_storex_abs(mach.sizeOf(i.op.typeArgs[1]), addr, ssaval);
+			if (t.typeCon.kind == V3Kind.FLOAT) gen_storex_abs_sse(mach.sizeOf(t), addr, ssaval);
```

Comment:
Similarly, this should probably use {isDouble()} and make it impossible to have an incorrect size. It might be better to do the if (isDouble) check here instead of lower down, too.